bears/vcs/CommitBear.py

```diff
@@ -203,23 +203,24 @@ def check_shortlog(self, shortlog,
                     'Shortlog of HEAD commit does not match given regex:'
                     ' {regex}'.format(regex=shortlog_regex))
 
+        if shortlog_wip_check:
+            if 'wip' in shortlog.lower()[:4]:
+                yield Result(
+                    self,
+                    'This commit seems to be marked as work in progress and '
+                    'should not be used in production. Treat carefully.')
+
         if shortlog_imperative_check:
             colon_pos = shortlog.find(':')
-            shortlog = (shortlog[colon_pos + 1:]
-                        if colon_pos != -1
-                        else shortlog)
-            has_flaws = self.check_imperative(shortlog)
+            shortlog_imp = (shortlog[colon_pos + 1:]
```

Comment:
you dont need to change the name of this variable.

If anyone wants to see the real complete shortlog, theor code should be placed above this.

Or they can create a new variable.